confgenerator/loggingmetrics.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package confgenerator
```

Comment:
It's a bit weird but could we put this in `../apps/fluentbit.go` perhaps?

Reply:
I'm currently exploring a different implementation that would add this as an app instead of a default configuration. we can sync on that later tomorrow.

Reply:
I took a look at this, and it creates a cyclic dependency between confgenerator and apps packages. I think if we want to move this (and possibly the very similar agentmetrics.go) out of here, we would need to refactor the code a bit.

Reply:
No worries, we can postpone cleaning this up.